home/decorators.py

```diff
@@ -12,3 +12,18 @@ def wrapper(request, *args, **kwargs):
         return wrapper
 
     return user_in_group
+
+
+def group_required_multiple(group, group_two):
```

Comment:
There's got to be a way to generalize this to work for an arbitrary list of groups not just two. Probably something like:

```python
request.user.groups.filter(name__in=groups).exists()
```

https://docs.djangoproject.com/en/2.2/ref/models/querysets/#in

Reply:
Taken care of with:

`def group_required_multiple(group_list):
    def user_in_group(func):
        def wrapper(request, *args, **kwargs):
            group = [group for group in request.user.groups.all() if group in group_list]
            if group is not None:
                return func(request, *args, **kwargs)
            else:
                raise PermissionDenied

        return wrapper

    return user_in_group
`